Formula/p/pyenv.rb

```diff
@@ -1,8 +1,8 @@
 class Pyenv < Formula
   desc "Python version management"
   homepage "https://github.com/pyenv/pyenv"
-  url "https://github.com/pyenv/pyenv/archive/refs/tags/v2.3.24.tar.gz"
-  sha256 "3a256f1026644f4ba79812fc7242e0342de5c4d12c6cbf2214c7d1af232e82df"
+  url "https://github.com/pyenv/pyenv/archive/refs/tags/v2.3.35.tar.gz"
```

Comment:
It seems that upstream mistakenly tagged `v2.3.35` while it should be `v2.3.25`.